tests/test_room_config.py

```diff
@@ -0,0 +1,266 @@
+"""
+Tests for room configuration loading and validation
+"""
+
+import os
+import tempfile
+import pytest
+from pathlib import Path
+
+from src.commons.room_config import (
+    load_rooms_config,
+    get_room_by_id,
+    get_enabled_rooms,
+    rooms_to_dict,
+    RoomConfig,
+    RoomsConfiguration,
```

Comment:
it's somewhere around this point that I would normally switch to just `from src.commons import room_config as rooms`